workflows/publish-monorepo.sh

```
@@ -113,21 +114,23 @@ detect_changed_packages() {
-      manifest_key="packages/$package"
+      if [[ " ${providers[*]} " =~ " ${package} " ]]; then
+        manifest_key="packages/providers/$package"
+      else
+        manifest_key="packages/$package"
+      fi
+
       expected_version=$(jq --raw-output ".[\"$manifest_key\"]" .release-please-manifest.json)
 
       # Compare versions to detect changes
-      if [ "$current_version" != "$expected_version" ]; then
+      if [[ "$current_version" != "$expected_version" || "$forcePublish" == "1" ]]; then
```

Shellcheck (0.10.0)

[warning] forcePublish is referenced but not assigned.

(SC2154)